ihatemoney/migrations/versions/afbf27e6ef20_add_bill_import_date_field.py

```diff
@@ -0,0 +1,26 @@
+"""add bill.import_date field
+
+Revision ID: afbf27e6ef20
+Revises: b78f8a8bdb16
+Create Date: 2018-02-19 20:29:26.286136
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'afbf27e6ef20'
+down_revision = 'b78f8a8bdb16'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('bill', sa.Column('import_date', sa.Date(), nullable=True))
```

Comment:
Maybe I should fill the new field to mimmick the `date` value for already existing bills?